contracts/contracts/tokenBridge/interfaces/ITokenBridge.sol

```diff
@@ -214,6 +214,11 @@ interface ITokenBridge {
    */
   error DecimalsAreUnknown(address token);
 
+  /**
+   * @dev Token list is empty.
```

Comment:
Please align this natspec message with the other natspec error messages :)

Reply:
done